public/bundle.js

```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
GET THIS OUT OF HERE

Reply:
Ok, all done.  You can approve now, pleeeease!!!!